public/components/query_compare/search_result/result_components/result_grid.tsx

```diff
@@ -44,17 +44,19 @@ export const ResultGridComponent = ({
     );
   };
 
-  const getDlTmpl = (doc: IDocType) => {
+  const getDlTmpl = (doc: Document) => {
+    const sourceFields = Object.assign(doc._source, doc.fields);
+
     return (
       <div className="truncate-by-height">
         <span>
           <dl className="source truncate-by-height">
-            {_.toPairs(doc).map((entry: string[]) => {
+            {_.toPairs(sourceFields).map((entry: string[]) => {
               return (
                 <>
                   <dt>{`${entry[0]}:`}</dt>
                   <dd>
-                    <span>{_.isObject(entry[1]) ? JSON.stringify(entry[1]) : entry[1]} </span>
```

Comment:
Any reason why this space was removed? May need a space in between entries, right?

Reply:
Good catch! I inadvertently deleted it while refactoring the component.